scenarios/arcaflow/cpu-hog/config.yaml

```diff
@@ -0,0 +1,10 @@
+logged_outputs:
+  error:
+    level: error
+  success:
+    level: debug
+deployer:
+  type: podman
```

Comment:
should we add kubernetes as the base deployer here?

Reply:
If I understand properly kubernetes will use the kuberenetes api with the kubeconfig we set in the main config... Is that what we want?